synthtool/gcp/templates/python_library/README.rst

```diff
@@ -0,0 +1,90 @@
+Python Client for {{ metadata['repo']['name_pretty'] }} API
+==============================================
+
+|{{ metadata['repo']['release_level'] }}| |pypi| |versions|
+
+`{{ metadata['repo']['name_pretty'] }} API`_: CHANGE ME: ADD API DESCRIPTION HERE
```

Comment:
Instead of the `CHANGE ME`, lets instead use `metadata['description']` here.  You can choose to only add it if the desc field is available.